.github/workflows/ci.yml

```
@@ -38,7 +38,7 @@ jobs:
               baseRef = 'origin/' + baseRef;
             }
             const child_process = require( 'node:child_process' );
-            child_process.execSync( 'pnpm utils ci-jobs ' + baseRef );
+            child_process.execSync( `pnpm utils ci-jobs ${ baseRef ? `--base-ref ${ baseRef }` : '' }` );
```

This would be cleaner if line 38 was changed to

```
baseRef  = '--base-ref origin/' + baseRef;
```

Then this line could be

```suggestion
            child_process.execSync( `pnpm utils ci-jobs ${ baseRef  }` );
```

---

Good suggestion. Updated!